ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -57,27 +53,6 @@ import { useAccountTotalFiatBalance } from '../../../hooks/useAccountTotalFiatBa
 const MAXIMUM_CURRENCY_DECIMALS = 3;
 const MAXIMUM_CHARACTERS_WITHOUT_TOOLTIP = 17;
 
-function getLabel(t, { type }) {
```

Comment:
This method has been replaced by `getAccountLabel`. I only see `getAccountLabel` gets called inside a code fence but the `getLabel` account was not fenced. Does this mean we are missing a use case for `getAccountLabel`?

Reply:
This is a good point and I believe we can remove the code fencing around the larger function